packages/api-db-mongodb/src/migration.ts

```diff
@@ -94,6 +95,40 @@ export const Migrations: Migration[] = [
         strict: true
       })
     }
+  },
+  {
+    version: 1,
+    async migrate(db, locale) {
+      const userRoles = await db.createCollection<v1.DBUserRole>(v1.CollectionName.UserRoles, {
+        strict: true
+      })
+
+      await userRoles.createIndex({name: 1}, {unique: true})
+
+      userRoles.insertMany([
```

Comment:
@per2plex not sure where to insert entries like that. I guess it makes sense here but that means this has to be copied to the other db adapters as well. What do you think?

Reply:
I think this is fine for now. Another way would be to standardize the initialization process. But I also thought about abandoning DB adaptability in the future in favor of using MongoDB directly which would allow for easier pluggability.